release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1376,6 +1376,11 @@ In the following tables, features are marked with the following statuses:
 |Not Available
 |Technology Preview
 
+|SR-IOV Operator support for Intel C741 Emmitsburg Chipset
```

Comment:
```suggestion
|SR-IOV Network Operator support for Intel C741 Emmitsburg Chipset
```

The official Operator name is SR-IOV Network Operator

Reply:
Thanks @cgoncalves for your feedback. I have applied your suggestion.

I know that merge is on hold for this story. Are the Tech Preview details still being finalised internally?

Reply:
LGTM. Yes, still being finalised internally.